arc-script/arc-script-core/src/compiler/ast/display.rs

```diff
@@ -396,7 +396,6 @@ impl<'i> Display for Pretty<'i, ast::BinOp, Context<'_>> {
             ast::BinOpKind::Leq  => write!(f, " <= "),
             ast::BinOpKind::Or   => write!(f, " or "),
             ast::BinOpKind::And  => write!(f, " and "),
-            ast::BinOpKind::Xor  => write!(f, " xor "),
```

Comment:
Actually we could support xor for booleans quite simply if you want it. The code generator would just transform  `(A xor B)` to `((A == !B) || (!A == B))`

Reply:
Or, as booleans are `i1` in MLIR, you could just use the bxor, the result is the same.

Reply:
Interesting, then I might add it back.

Reply:
Is it `arc.xor` or `std.bxor`?

Reply:
`std.xor`: https://mlir.llvm.org/docs/Dialects/Standard/#stdxor-xorop